File: jqwik-starter-maven/src/test/java/my/example/project/MyCalculatorExamples.java
package my.example.project;

import net.jqwik.api.*;
import org.assertj.core.api.*;

import static java.util.Arrays.*;

class MyCalculatorExamples {

	@Example
	void summingUpZeros() {
		int result = new MyCalculator().sum(asList(0, 0, 0));
		Assertions.assertThat(result).isEqualTo(0);
	}

	@Example
	void summingUpTwoNumbers() {
		int result = new MyCalculator().sum(asList(1, 41));
		Assertions.assertThat(result).isEqualTo(42);
	}
}

File: jqwik-starter-gradle-android/src/test/java/my/example/project/android/MyCalculatorExamples.kt
package my.example.project.android

import net.jqwik.api.Example

class MyCalculatorExamples {
    @Example
    fun summingUpZeros() {
        val result = MyCalculator().sum(0, 0, 0)
        assert(result == 0)
    }

    @Example
    fun summingUpTwoNumbers() {
        val result = MyCalculator().sum(1, 41)
        assert(result == 42)
    }
}